models/UserSearch.php

```diff
@@ -93,6 +97,11 @@ public function search($params)
             return $dataProvider;
         }
 
+        if ($this->auth_items) {
+            $query->leftJoin('auth_assignment', 'auth_assignment.user_id = user.id');
```

Comment:
These table names are not guaranteed. Instead you should use the `ManagerInterface` interface to get permissions or assignments.
http://www.yiiframework.com/doc-2.0/yii-rbac-managerinterface.html

I can implement RBAC from a database, a text file or any other backend.